### ISSUE TITLE
Full text index is not updated field update

### ISSUE THREAD

**Issue by angelix** [dizitart/nitrite-database#222]
Hi @anidotnet,

On latest version (3.4.1) seems that when i change the value of text indexed field, the old value is not removed from the index. So using Filters.text operator i get the document for the old and new value.

Reproduce:
```
1. create Text index on a field
2. insert document with text index field
3. search document with Filters.text() -> Works OK
4. update document, removing the text from the indexed field, so now 
5. search document with Filters.text() -> keeps returning the document.
```

Thanks

**Comment by anidotnet** [comment]
Hi @angelix is it happening for you in the previous versions or it just appeared only in 3.4.1?

**Comment by angelix** [comment]
I quickly checked with previous version (3.3.0) and have the same behavior, I'll double check to be sure.

**Comment by angelix** [comment]
Seems that the text indexes are never removed or updated.

**Comment by anidotnet** [comment]
@angelix are you using the Lucene based test indexing example given in the test code?

**Comment by angelix** [comment]
@anidotnet no, i am using the nitrite's one.

**Comment by angelix** [comment]
@anidotnet do you want a code sample?

**Comment by anidotnet** [comment]
Sorry @angelix I am busy with my day job. I'll work on it as soon as I get some time. Meantime, if you can provide a code sample it would be great.

**Comment by anidotnet** [comment]
The fix will be available in 3.4.2
